gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -68,4 +103,11 @@ private static String getStringOrEmpty(
     final String value = request == null ? null : valueExtractor.apply(request);
     return value == null ? "" : value;
   }
+
+  private static String getStringOrEmpty(
```

Comment:
🔧 Can we generalize the extraction method we already have instead to take any request? Let's avoid code duplication here. The value extractor function and the request should be of the same generic type.